docs/writing-rules/testing-rules.md

```diff
@@ -31,6 +31,27 @@ Other than annotations there are three things to remember when creating tests:
 The `.test.yaml` file extension can also be used for test files. This is necessary when testing YAML language rules.
 :::
 
+## Testing autofix
+
+Semgrep's testing mechanism also provides a way to test the behaviour of any fixes defined in the rules.
```

Comment:
```suggestion
Semgrep's testing mechanism also provides a way to test the behavior of any `fix` values defined in the rules.
```

Just making it clear that "fixes" was referring to the `fix` (rule syntax) values.